tests/cache/test_redis_backend.py

```diff
@@ -149,6 +149,7 @@ def test_connect_with_defaults(self, MockStrictRedis):
         # The defaults, used if keys are missing from the arguments dict.
         arguments = {
             "host": "localhost",
+            "username": None,
```

Comment:
can we omit username here?   existing code that doesnt send a username in should continue to work

Reply:
I followed how password is tested but I agree testing the actual omit case is better. Updated to test without these optional arguments